src/openrct2/peep/Peep.cpp

```diff
@@ -2732,9 +2732,10 @@ static void peep_footpath_move_forward(Peep* peep, int16_t x, int16_t y, TileEle
  *
  *  rct2: 0x0069455E
  */
-static void peep_interact_with_path(Peep* peep, int16_t x, int16_t y, TileElement* tile_element)
+static void peep_interact_with_path(Peep* peep, const CoordsXYE& coords)
 {
     // 0x00F1AEE2
+    auto* tile_element = coords.element;
```

Comment:
No need for the `*` when the auto is deducing a pointer, it can deduce this just fine. You only ever need to type either `auto` or `auto&`
```suggestion
    auto tile_element = coords.element;
```